src/main/java/ch/uzh/ifi/hase/soprafs21/entity/Tag.java

```diff
@@ -0,0 +1,45 @@
+package ch.uzh.ifi.hase.soprafs21.entity;
+
+import ch.uzh.ifi.hase.soprafs21.constant.TagType;
+import lombok.*;
+
+import javax.persistence.*;
+
+/**
+ * Internal Tag representation
+ */
+@Data
+@EqualsAndHashCode(exclude="owner")
+@Builder
+@Entity
+@NoArgsConstructor
+@AllArgsConstructor
+@NamedEntityGraph(name = "tags_without_owner",
+        includeAllAttributes = true,
```

Comment:
What is the reason to use graph in a Tag entity?

Reply:
removed in commit def59ab

Reply:
I asked because I didn't understand the reason. Was it just to get TagDto without the owner attribute? If this is the case, you do not map it in the TagsMapper(this is already enough) or you can add @JsonIgnore annotation to owner.

Reply:
Thanks for your explanation! Sounds reasonable.